acbuild/copy.go

```diff
@@ -20,9 +20,9 @@ import (
 
 var (
 	cmdCopy = &cobra.Command{
-		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
+		Use:     "copy PATH_ON_HOST... PATH_IN_ACI",
```

Comment:
Could split the different behaviors out into different commands, but I'd want to be careful about naming to make sure it's clear why there are two separate commands. I'm still not sure why Dockerfiles have both Add and Copy. 

> On Nov 12, 2015, at 18:13, Jonathan Boulle <notifications@github.com> wrote:
> 
> In acbuild/copy.go:
> 
> > @@ -20,9 +20,9 @@ import (
> >  
> >  var (
> >  	cmdCopy = &cobra.Command{
> > -		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
> > +		Use:     "copy PATH_ON_HOST... PATH_IN_ACI",
> I'm a little torn here; it would be great to be explicit about these different operation modes. Any other ideas?
> 
> ―
> Reply to this email directly or view it on GitHub.
>